graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/internal/InputArgumentTest.kt

```diff
@@ -1467,6 +1457,51 @@ internal class InputArgumentTest {
         verify { applicationContextMock.getBeansWithAnnotation(DgsComponent::class.java) }
     }
 
```

Comment:
Do we already have a test case covering the nested enum type use case?

Reply:
We have plenty of tests that include _enums_ as part of other complex types but not directly as an input argument.